src/api/submission/writer.py

```diff
@@ -153,9 +153,15 @@ def get_or_create_fishbelt_transect(self, sample_event_id):
             FishBeltTransect, FishBeltTransectSerializer, fishbelt_transect_data
         )
 
-    def get_or_create_beltfish(self, fishbelt_transect_id, sample_unit_method_id=None):
+    def get_or_create_beltfish(
+        self, collect_record_id, fishbelt_transect_id, sample_unit_method_id=None
+    ):
         beltfish_data = {"transect": fishbelt_transect_id, "id": sample_unit_method_id}
-        return self.get_or_create(BeltFish, BeltFishSerializer, beltfish_data)
+        record = self.get_or_create(BeltFish, BeltFishSerializer, beltfish_data)
+        record.collect_record_id = collect_record_id
+        record.save()
```

Comment:
Are we adding `collect_record_id` to support SU->CR conversion? I thought we already did that for fishbelt?

Can we not avoid saving twice by setting `collect_record_id` before doing `get_or_create`? I worry about signals and whatnot...

Reply:
Agreed, pushed up a change